archive_viewer/mixins/plot_config.py

```diff
@@ -0,0 +1,66 @@
+from widgets import ColorButton
+from pyqtgraph import ViewBox
+from typing import Dict
+from qtpy.QtGui import QColor
+from time import sleep
+
+class PlotConfigMixin:
```

Comment:
Style: Can you add docstrings to the class and all the methods in this class? thanks